RecoTracker/TkDetLayers/src/TBLayer.h

```diff
@@ -25,14 +25,14 @@ class TBLayer: public BarrelDetLayer {
 
   // GeometricSearchDet interface
   
-  virtual const std::vector<const GeomDet*>& basicComponents() const final {return theBasicComps;}
+  virtual const std::vector<const GeomDet*>& basicComponents() const {return theBasicComps;}
```

Comment:
is removal of finalization justified?
I see there is a formal overload (in an apparently derived from this) Phase2OTtiltedBarrelLayer, but the overload uses the same ``` return theBasicComps ```, except that those theBasicComps are also a data member in the derived